src/roadmap.md

```diff
@@ -91,7 +91,7 @@ We already started the implementation of Java binding.
 
 ## Interface Description
 
-Exonums 0.1 requires that a service developer manually specify a number of
+Exonum 0.1 requires that a service developer manually specify a number of
```

Comment:
"that a service developer manually specify" --> "that a service developer should manually specify"